app/build.gradle.kts

```diff
@@ -55,10 +53,6 @@ android {
 	}
 }
 
-configure<NoArgExtension> {
-	annotation("com.intive.picover.common.annotation.NoArgConstructor")
```

Comment:
Used for generation default and empty constructor for `PartyRemote`, to handle call `childSnapshot.getValue(PartyRemote::class.java)`

Reply:
Nice!  😄